manager/testdata/new-blueprints/scenario2/arrow-flight-module.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: app.m4d.ibm.com/v1alpha2
+kind: M4DModule
+metadata:
+  name: arrow-flight-read
+  namespace: m4d-system
+  labels:
+    version: 0.0.1
+    name: arrow-flight-read
+spec:
+  description: "This module deploys an arrow-flight service"
+  # this field is true if this module serves multiple users. Otherwise it
+  # is false. It is used for example when setting the endpoint to
+  # access the module service - if it is a multi-user module then
+  # the service name will be fixed and will be taken from
+  # M4DModule.Capabilities.Api.endpoint.host.
+  type: service
+  confChart:
+    name: ghcr.io/mesh-for-data/read-module:0.1.0
+    values: {}
+  statusIndicators:
+  - kind: Deployment
+    successCondition: status == SUCCEEDED # TBD with correct values
+    failureCondition: status == FAILED # TBD with correct values
+    errorMessage: status.error # TBD with correct values
+  capabilities:
+  - capability: read
+    scope: workflow
+    sources:
+    - protocol: s3
+      dataformat: parquet
+    - protocol: s3
+      dataformat: csv
+    plugins:
+      - protocol: fybrik-action
```

Comment:
@froesef Don't we need to declare the plugin capabilities supported?  I had envisioned ...
plugins:
- capability: transform
   protocol: fybrik-wasm-plugin
   dataformat: arrow

Reply:
All plugins deployed here are transform as the capability above is transform.

Reply:
Remove protocol and format.  Put plugin type instead - ex: wasm, vault